lib/galaxy/quota/_schema.py

```diff
@@ -16,10 +20,12 @@
     UserModel,
 )
 
-QUOTA_MODEL_CLASS_NAME = "Quota"
-USER_QUOTA_ASSOCIATION_MODEL_CLASS_NAME = "UserQuotaAssociation"
-GROUP_QUOTA_ASSOCIATION_MODEL_CLASS_NAME = "GroupQuotaAssociation"
-DEFAULT_QUOTA_ASSOCIATION_MODEL_CLASS_NAME = "DefaultQuotaAssociation"
+
+class ModelClassName(str, Enum):
+    QUOTA = "Quota"
+    USER_QUOTA_ASSOCIATION = "UserQuotaAssociation"
+    GROUP_QUOTA_ASSOCIATION = "GroupQuotaAssociation"
+    DEFAULT_QUOTA_ASSOCIATION = "DefaultQuotaAssociation"
```

Comment:
Why not just `ModelClassName = Literal["Quota", "UserQuotaAssociation", "GroupQuotaAssociation", "DefaultQuotaAssociation"]` ?

I think that would simplify the code and make it much more readable by just using strings below.

Reply:
In that case I don't know what the `ModelClassName ` would buy us. I could just move this into the actual database model class, or use strings.

Reply:
Just re-reading Nicola's suggestion I noticed, we must explicitly set one of those literals for each model, so there is no point in having the ModelClassName as Marius pointed out, we can directly use the string in the corresponding model if the enum is ugly.
I personally don't like to use plain strings because I kind of consider them magic words and I prefer the IDE to autocomplete and validate them for me to avoid typos. Sooo... up to you Marius :laughing: